### PR TITLE
feat: base price on category

### PR DESCRIPTION
**Pull request by rbedeoan-plenty**
https://dev.azure.com/plentymarkets/plentymarkets/_sprints/backlog/BD%20-%20Team%20-%20LM/plentymarkets/Areas/Sprint%2033?workitem=63854

### CODE REVIEW

**Comment by rbedeoan-plenty** [comment]
we should use product getters from productGetters not cartGetters...

**Comment by rbedeoan-plenty** [comment]
@maxiroellplenty 
2 questions
1.are you sure about this request: 
"Try to unify changes by having reusable components (LTS uses the same item-component for basket-preview, Basket and checkout >> basketListItem.vue" 
Both files in which cart product list is used have diferent ways of displaying the product list (table style..)
2.should't we use getter function only from productGetters when whe display product fields? (for a new dev that interacts with the app seems more straight forward..)

**Comment by rbedeoan-plenty** [comment]
currently failing paypal e2e,i think it does't have to do with changes in current branch

File: packages/theme/components/BasePrice.vue
Comment (FabianGerke) [inline packages/theme/components/BasePrice.vue:1]: Is there a way just to give the product-prop to the component?

File: packages/theme/components/CartSidebar.vue
Comment (FabianGerke) [inline packages/theme/components/CartSidebar.vue:1]: Not really consistent

File: packages/theme/pages/Product.vue
Comment (FabianGerke) [inline packages/theme/pages/Product.vue:1]: Maybe we can do it like:

<BasePrice :product="product" class="baseprice-online" />

?

File: packages/theme/components/BasePrice.vue
Comment (FabianGerke) [inline packages/theme/components/BasePrice.vue:1]: Use getUnitContent instead of getUnitId

File: packages/theme/components/BasePrice.vue [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    }
+
+    return {
+      basePriceCss,
+      productGetters
+    };
+  }
+};
+</script>
+<style scoped>
```
Comment (FabianGerke) [inline packages/theme/components/BasePrice.vue:64]: Remove some css with tailwind

**Review by FabianGerke — CHANGES_REQUESTED** [review]

File: packages/theme/components/BasePrice.vue [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  productGetters
+} from '@vue-storefront/plentymarkets';
+export default {
+  props: {
+    product: {
+      type: Object,
+      default: () => { }
+    },
+    oneline: {
+      type: Boolean,
```
[the remark this answers is not in the document]
Comment (rbedeoan-plenty) [reply packages/theme/components/BasePrice.vue:42]: need some context

**Review by FabianGerke — DISMISSED** [review]

**Review by FabianGerke — APPROVED** [review]

**Pull request merged**